com.gitlab.bitseater.meteo.json

```diff
@@ -0,0 +1,148 @@
+{
+    "app-id" : "com.gitlab.bitseater.meteo",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.gitlab.bitseater.meteo",
+    "build-options" : {
+        "cflags" : "-O2  -Wno-deprecated-declarations",
```

Comment:
The -O2 flags are not needed.

Reply:
Hi @bilelmoussaoui . Thanks for your support. -O2 flags only in line 8 . Can i delete line 9 too?
`        "cxxflags" : "-O2",`?

Reply:
Ok. i just see the comments by @nedrichards about this.